app/src/main/java/com/iet/ietians_diary/DashboardFragment.java

```diff
@@ -98,10 +98,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         newFeatures.setNestedScrollingEnabled(false);
 
         ArrayList<DashboardNewFeaturesModel> list1 = new ArrayList<>();
-        list1.add(new DashboardNewFeaturesModel("Discussion Forum", "Lorem Ipsum is simply dummy \ntext of the printing and \ntypesetting industry.", R.drawable.feature_discussion_forum_bg_blue, R.drawable.feature_discussion_forum_textview_bg_blue));
-        list1.add(new DashboardNewFeaturesModel("Interview", "Lorem Ipsum is simply dummy \ntext of the printing and \ntypesetting industry.", R.drawable.feature_discussion_forum_bg_green, R.drawable.feature_discussion_forum_textview_bg_green));
-        list1.add(new DashboardNewFeaturesModel("Doubts", "Lorem Ipsum is simply dummy \ntext of the printing and \ntypesetting industry.", R.drawable.feature_discussion_forum_bg_grey, R.drawable.feature_discussion_forum_textview_bg_grey));
-        list1.add(new DashboardNewFeaturesModel("Upload", "Lorem Ipsum is simply dummy \ntext of the printing and \ntypesetting industry.", R.drawable.feature_discussion_forum_bg_red, R.drawable.feature_discussion_forum_textview_bg_red));
+        list1.add(new DashboardNewFeaturesModel("Discussion Forum", "Coming Soon", R.drawable.feature_discussion_forum_bg_blue, R.drawable.feature_discussion_forum_textview_bg_blue));
```

Comment:
Please don't remove the description. Create another text field below the description and put `coming soon` there.